portfolio/src/main/webapp/script.js

```diff
@@ -56,23 +56,49 @@ function loadSkills () {
  */
 async function fetchFromData() {
   const response = await fetch('/data');
+  
   // This gives a list of messages
-  const msgs = await response.json();
+  const comments = await response.json();
 
   const root = document.getElementById("all-messages");
 
   // Adds all of the messages to a div
-  msgs.forEach(m => {
-    const node = document.createElement("p"); 
-    node.className = "msg"; 
-    node.textContent = m;
-    root.appendChild(node); 
+  comments.forEach(c => {
+    root.appendChild(createCommentElement(c)); 
   }); 
 
   $("#get-msg-btn").addClass("invisible"); 
   $("#hide-msg-btn").removeClass("invisible"); 
 };
 
+/**
+ * Creates a comment element with the content, name of commenter, and date
+ */
+function createCommentElement(comment) {
+  const node = document.createElement("div");
+  node.className = "msg"; 
+
+  const contentElement = document.createElement('span');
+  contentElement.className = "comment-content";
+  contentElement.innerText = comment.content;
+
+  const nameElement = document.createElement('p');
+  nameElement.className = "comment-name";
+  nameElement.textContent = `${comment.name} at ${parseTimeforDisplay(comment.timestamp)}`;
+
+  node.appendChild(contentElement);
+  node.appendChild(nameElement);
+  return node;
+}
+
+/**
+ * Takes in a long date representation and converts it to a more readable time
+ */
+function parseTimeforDisplay(time) {
```

Comment:
This method seems simple enough and is only used once -- consider inlining it

Reply:
AH yea this was mostly in case I wanted to do more with the date -- but I'm not 100% sure I will actually